File: back/php/testApp/counterparty.php
<?php

class Counterparty{
    // наименование
    public $name;
        
    // ИНН
    public $inn;

    // КПП
    public $kpp;

    // банк
    public $bank;

    // расчтеный счет
    public $rs;

    public function toString(){
        return iconv("utf-8", "windows-1251", "Наименование: ") . $this->name . 
             iconv("utf-8", "windows-1251", "<br>ИНН: ") . $this->inn . 
             iconv("utf-8", "windows-1251", "<br>КПП: ") . $this->kpp . 
             iconv("utf-8", "windows-1251", "<br>Банк: ") . $this->bank . 
             iconv("utf-8", "windows-1251", "<br>Расчётный счёт: ") . $this->rs; 
    }

}
?>

File: back/java/testApp/src/testapp/Counterparty.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package testapp;

/**
 * Контрагент
 *
 * @author ili4
 */
public class Counterparty {

    /**
     * Наименование
     */
    public String name;

    /**
     * ИНН
     */
    public String inn;

    /**
     * КПП
     */
    public String kpp;

    /**
     * Банк
     */
    public String bank;

    /**
     * Расчетный счет
     */
    public String rs;

    @Override
    public String toString() {
        return "Наименование: " + name + 
                "\nИНН: " + inn + 
                "\nКПП: " + kpp + 
                "\nБанк: " + bank + 
                "\nРасчётный счёт: " + rs;
    }
}
